osquery/distributed/distributed.cpp

```diff
@@ -181,7 +184,9 @@ Status Distributed::acceptWork(const std::string& work) {
                       "Distributed query does not have complete attributes.");
       }
       WriteLock wlock(distributed_queries_mutex_);
-      queries_.push_back(request);
+      setDatabaseValue(kQueries,
+        dist_query_prefix + node.first,
+        queries.get<std::string>(node.first, ""));
```

Comment:
If you're not using the `DistributedQueryRequest` object here, remove it. Also reconsider it's existence in the first place.